### ISSUE TITLE
OpenVPN Road Warrior: route issue with bond management address

### ISSUE THREAD

**Issue by gsanchietti** [NethServer/nethsecurity#931]
**Steps to reproduce**
- Configure the ln interface as a bond on NethSecurity.
- Automatically a /32 management address is assigned to the bond.
- Set up an OpenVPN roadwarrior server.
- Connect a roadwarrior client, such as OpenVPN Connect on an Android phone.

**Expected behavior**
- The /32 management IP address should not be included in the "push networks route" by default when creating the OpenVPN roadwarrior server.

**Actual behavior**
- The client receives the /32 IP address among the routes, causing OpenVPN Connect on Android to display an error message: "emulate_exclude_route_error: tunb_builder_add_route failed."

**Components**
NethSecurity version: 8-23.05.5-ns.1.3.0

**Comment by gsanchietti** [comment]
Image: [23.05.5-ns.1.3.0-52-ge672f104](https://github.com/NethServer/nethsecurity/actions/runs/11955776070/artifacts/2219331818)

**Test case**
Check the issue is not reproducibile

**Comment by francio87** [comment]
Confirm Fixed ✅ 

```sh
root@nsec-esxi:~# ip -c a sh bond-bondtest
8: bond-bondtest: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 00:0c:29:7c:62:ab brd ff:ff:ff:ff:ff:ff
    inet 10.99.123.5/24 brd 10.99.123.255 scope global bond-bondtest
       valid_lft forever preferred_lft forever
    inet 127.121.238.1/32 brd 255.255.255.255 scope global bond-bondtest
       valid_lft forever preferred_lft forever
```

The management IP of the bond is not included in the push route of the OpenVPN Server :


```sh
root@nsec-esxi:~# uci show openvpn.ns_roadwarrior1.push
openvpn.ns_roadwarrior1.push='route 10.99.123.0 255.255.255.0'
```

**Comment by gsanchietti** [comment]
Released in python3-nethsec-0.0.84
